module/data/menu_keys.py

```diff
@@ -0,0 +1,55 @@
+from telegram import InlineKeyboardButton
+
+"""
+    to do: make keys costants values
+"""
+"""
+GIORNI_PRANZO_CENA =    [
+                            InlineKeyboardButton("Giorni", callback_data="0"),
+                            InlineKeyboardButton("Pranzo", callback_data="0"),
+                            InlineKeyboardButton("Cena", callback_data="0")
+                        ]
+
+monday = InlineKeyboardButton("Lunedì", callback_data="0")
+tuesday = InlineKeyboardButton("Martedì", callback_data="0")
+wednesday = InlineKeyboardButton("Mercoledì", callback_data="0")
+thursday = InlineKeyboardButton("Giovedì", callback_data="0")
+friday = InlineKeyboardButton("Venerdì", callback_data="0")
+saturday =  InlineKeyboardButton("Sabato", callback_data="0")
+sunday = InlineKeyboardButton("Domenica", callback_data="0")
+
+monday_lunch_false = InlineKeyboardButton("❌", callback_data="1")
+monday_lunch_true = InlineKeyboardButton("✅", callback_data="2")
+monday_dinner_false = InlineKeyboardButton("❌", callback_data="3")
+monday_dinner_true = InlineKeyboardButton("✅", callback_data="4")
+
+tuesday_lunch_false = InlineKeyboardButton("❌", callback_data="5")
+tuesday_lunch_true = InlineKeyboardButton("✅", callback_data="6")
+tuesday_dinner_false = InlineKeyboardButton("❌", callback_data="7")
+tuesday_dinner_true = InlineKeyboardButton("✅", callback_data="8")
+
+wednesday_lunch_false = InlineKeyboardButton("❌", callback_data="9")
+wednesday_lunch_true = InlineKeyboardButton("✅", callback_data="10")
+wednesday_dinner_false = InlineKeyboardButton("❌", callback_data="11")
+wednesday_dinner_true = InlineKeyboardButton("✅", callback_data="12")
+
+thursday_lunch_false = InlineKeyboardButton("❌", callback_data="13")
+thursday_lunch_true = InlineKeyboardButton("✅", callback_data="14")
+thursday_dinner_false = InlineKeyboardButton("❌", callback_data="15")
+thursday_dinner_true = InlineKeyboardButton("✅", callback_data="16")
+
+friday_lunch_false = InlineKeyboardButton("❌", callback_data="17")
+friday_lunch_true = InlineKeyboardButton("✅", callback_data="18")
+friday_dinner_false = InlineKeyboardButton("❌", callback_data="19")
+friday_dinner_true = InlineKeyboardButton("✅", callback_data="20")
+
+saturday_lunch_false = InlineKeyboardButton("❌", callback_data="21")
+saturday_lunch_true = InlineKeyboardButton("✅", callback_data="22")
+saturday_dinner_false = InlineKeyboardButton("❌", callback_data="23")
+saturday_dinner_true = InlineKeyboardButton("✅", callback_data="24")
+
+sunday_lunch_false = InlineKeyboardButton("❌", callback_data="25")
+sunday_lunch_true = InlineKeyboardButton("✅", callback_data="26")
+sunday_dinner_false = InlineKeyboardButton("❌", callback_data="27")
+sunday_dinner_true = InlineKeyboardButton("✅", callback_data="28")
```

Comment:
all this code is really redundant we have to follow the DRY principle and wrap it into an array if we really need it
but I don't think that we need this